services/neutrino/neutrino.js

```diff
@@ -236,15 +242,29 @@ class Neutrino extends EventEmitter {
    * Set the lnd cfilter height and emit an event to notify others if it has changed.
    * @param {String|Number} height Block height
    */
-  setLndCfilterHeight(height) {
+  setNeutrinoCfilterHeight(height) {
     const heightAsNumber = Number(height)
     const changed = Neutrino.incrementIfHigher(this, 'neutrinoCfilterHeight', heightAsNumber)
     if (changed) {
+      mainLog.info('Set neutrino cfilter height', heightAsNumber)
       this.emit(NEUTRINO_GOT_LND_CFILTER_HEIGHT, heightAsNumber)
       this.setCurrentBlockHeight(heightAsNumber)
     }
   }
 
+  /**
+   * Set the current wallet recovery height and emit an event to notify others if it has changed.
+   * @param {String|Number} height Block height
+   */
+  setNeutrinoRecoveryHeight(height) {
+    const heightAsNumber = Number(height)
```

Comment:
Do we need to add `NaN` check just in case ?

Reply:
I added an `isNaN` check in `incrementIfHigher` method.